### PR TITLE
[CAI-41]: ChatBot FAB

### PR DESCRIPTION
**Pull request by Sebastiano-Bertolin**
#### List of Changes
Add ChatBot Floating Action Button

#### Motivation and Context

This button will open the chatbot modal to continue or start a new a chat

#### How Has This Been Tested?

It has been tested via Storybook

#### Screenshots (if appropriate):
Screenshot 2024-07-02 at 17 31 22
Screenshot 2024-07-02 at 17 31 55

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tommaso1 — APPROVED** [review]

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/atoms/ChatButton/ChatButton.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { Badge, Box } from '@mui/material';
+
+type ChatButtonProps = {
+  chatOpen: boolean;
+  hasNewMessages: boolean;
+  onOpenChat: (event: React.MouseEvent<HTMLButtonElement>) => null;
+};
+
+const ChatButton = ({
+  chatOpen,
```
[the remark this answers is not in the document]
Comment (Sebastiano-Bertolin) [reply apps/nextjs-website/src/components/atoms/ChatButton/ChatButton.tsx]: Done

File: apps/storybook-app/stories/atoms/ChatbotButton.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const meta: Meta<typeof ChatButton> = {
+  title: 'Atoms/ChatButton',
+  component: ChatButton,
+};
+
+export default meta;
+
+export const Showcase: StoryObj<typeof ChatButton> = {
+  args: {
+    chatOpen: false,
```
Comment (MarcoPonchia) [inline apps/storybook-app/stories/atoms/ChatbotButton.stories.tsx:13]: `chatOpen` args looks like is never used, maybe did  you mean to use `isChatOpen` ? :thinking:

File: apps/storybook-app/stories/atoms/ChatbotButton.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    onOpenChat: () => null,
+  },
+};
+
+export const ChatButtonNewMessages: StoryObj<typeof ChatButton> = {
+  args: {
+    chatOpen: false,
```
Comment (MarcoPonchia) [inline apps/storybook-app/stories/atoms/ChatbotButton.stories.tsx:20]: same here

Comment (Sebastiano-Bertolin) [reply apps/storybook-app/stories/atoms/ChatbotButton.stories.tsx]: Fixed

**Review by MarcoPonchia — APPROVED** [review]

File: .changeset/breezy-bulldogs-vanish.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
+"storybook-app": patch
```
Comment (marcobottaro) [inline .changeset/breezy-bulldogs-vanish.md:3]:
```suggestion
"nextjs-website": minor
"storybook-app": minor
```

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**